packages/wd-base/src/_demo.scss

```diff
@@ -0,0 +1,166 @@
+@import "colors";
```

Comment:
Can you rename this `_demo.scss` to `demo.scss` it's not a fragment, the outcome of it will generate a .css file.